src/EnergyPlus/ZoneTempPredictorCorrector.hh

```diff
@@ -336,6 +461,9 @@ struct ZoneTempPredictorCorrectorData : BaseGlobalStruct
     int IterLimitExceededNum2 = 0;
     int IterLimitErrIndex2 = 0;
 
+    EPVector<ZoneTempPredictorCorrector::ZoneHeatBalanceData> zoneHeatBalance;
+    EPVector<ZoneTempPredictorCorrector::SpaceHeatBalanceData> spaceHeatBalance;
```

Comment:
New workhorse heat balance vectors. AKA `thisZoneHB` and `thisSpaceHB` nearly everywhere.